docs/charts/area.md

```diff
@@ -32,6 +32,24 @@ new Chart(ctx, {
 });
 ```
 
+If you need to support multiple colors when filling from one dataset to another, you may specify an object as shown in the following:
+**Example**
+```javascript
+new Chart(ctx, {
+    data: {
+        datasets: [
+            {
+              fill: {
+                target: 'origin',
```

Comment:
I think this could use some additional documentation. E.g. what is `target` and what are the valid values that a user may pass to it?